control-plane/plugin/src/resources/pool.rs

```diff
@@ -577,12 +634,14 @@ impl Cordoning for Pool {
                     if resources.is_empty() {
                         println!("Pool {id} already uncordoned");
                     } else {
-                        println!("Pool {id} already partially uncordoned. Remaining cordoned resources: {resources:?}");
+                        println!("Pool {id} already partially uncordoned. Remaining constraints: {resources:?}");
```

Comment:
`println!("Pool {id} is partially uncordoned. Remaining constraints: {resources:?}");` ?